pkg/stub/handler.go

```diff
@@ -46,24 +46,32 @@ const (
 	skippedtempskey        = "keyForSkippedTemplatesField"
 )
 
-func NewHandler() sdk.Handler {
-	h := Handler{}
+func NewSamplesOperatorHandler(kubeconfig *restclient.Config) (*Handler, error) {
+	h := &Handler{}
 
-	h.initter = &defaultInClusterInitter{h: &h}
+	h.initter = &defaultInClusterInitter{h: h, restconfig: kubeconfig}
 	h.initter.init()
 
-	h.sdkwrapper = &defaultSDKWrapper{h: &h}
+	crdWrapper := &generatedCRDWrapper{}
+	crdWrapper.h = h
+	client, err := sampopset.NewForConfig(kubeconfig)
+	if err != nil {
+		return nil, err
+	}
+	crdWrapper.c = client.SamplesResources()
+
+	h.crdwrapper = crdWrapper
```

Comment:
handler contains crdwrapper which contains handler?  Seems like there's probably a better way to structure this?  Does the wrapper actually need the entire "handler" for something?

Reply:
A convention that has existed for a bit with most of these wrappers.

Now that the implementation has been sitting for a bit, I can replace the handler refs with the precise pieces within the handler that the wrapper needs.

Reply:
OK just pushed a separate commit to address this